official/resnet/keras/keras_common.py

```diff
@@ -419,8 +375,10 @@ def data_delay_prefetch():
 
 
 def set_cudnn_batchnorm_mode():
-  """Set CuDNN batchnorm mode for better performance. Note that the spatial
-     persistent mode may lead to accuracy losses for certain models."""
+  """Set CuDNN batchnorm mode for better performance. Note that the spatial.
```

Comment:
Don't split the sentence starting with "Note that the spatial". Instead, format like:

```
"""Set CuDNN batchnorm mode for better performance.

Note that the spatial persistent mode may lead to accuracy losses for certain
models.
"""

Reply:
Totally not thinking on my part.

Reply:
fixed